src/pages/images/ImageSelector.tsx

```diff
@@ -170,7 +170,7 @@ const ImageSelector: FC<Props> = ({ onClose, onSelect }) => {
             onClick: selectImage,
           },
           {
-            content: item.release,
+            content: item.release_title,
```

Comment:
By and large this should work, because we already have the shortname in the alias column, but that's not always the case, e.g.:

![image](https://github.com/canonical/lxd-ui/assets/56583786/c4184ef3-119c-4bd7-ac57-d93b33f21afd)

I think the original request from @lathiat was to show both. In this case the shortname "jammy" is not shown, could that be a problem?

Reply:
He mentioned juju moved away from the short name to the Version plus LTS. I think the current way is good, showing both in the column would break into two lines and looks not right on smaller screens.
I think all users will be able to pick the right release from the version and LTS string.

Reply:
I read:

> Ideally it would **also** display the release version

and:

> Ideally we'd have at a minimum consistency but realistically **just display both in as many scenarios as possible**.

Which is not what we're doing here, so if that's not fixing the original issue, it would be better to know it in advance. Let's wait a confirmation from the OP.

Reply:
Iterating on this a bit more - we can use the empty variant column for displaying the code name. I think it looks best this way, though we are bending the meaning of columns a bit. But seems like the best compromise to 
1. Show both the version and the short name
2. Show them in correct order like Ubuntu 22.04 LTS jammy
3. Avoid spacing issues on smaller screens